### PR TITLE
Swiftify API feature fixes

### PR DESCRIPTION
**Pull request by AnastasiaKubova**
Things to consider before you submit the PR:

* [x] Has `CHANGELOG.md` been updated?
* [x] Are tests passing locally?
* [x] Are the files formatted correctly?
* [ ] ~Did you add unit tests?~
* [x] Did you check UI tests on the sample app? They are not executed on CI.
* [x] Did you test your change with either the sample apps that are included in the repository or with a blank app that uses your change?

## Description

Feedback on the feature.

### CODE REVIEW

**Review by russelarms — APPROVED** [review]

File: AppCenter/AppCenter/Internals/Ingestion/MSACIngestionProtocol.h [hunk 1]
```diff
@@ -20,7 +20,7 @@ typedef void (^MSACSendAsyncCompletionHandler)(NSString *callId, NSHTTPURLRespon
 /**
  * The indicator of readiness to send data.
  */
-@property(nonatomic, readonly, getter=isReadyToSend) BOOL readyToSend NS_SWIFT_NAME(readyToSend);
+@property(nonatomic, readonly, getter=isReadyToSend) BOOL readyToSend;
```
Comment (MatkovIvan) [inline AppCenter/AppCenter/Internals/Ingestion/MSACIngestionProtocol.h:23]: Why not
```suggestion
@property(nonatomic, readonly) BOOL isReadyToSend;
```
?

File: AppCenterDistribute/AppCenterDistribute/Model/MSACReleaseDetails.h [hunk 2]
```diff
@@ -38,7 +38,7 @@ NS_SWIFT_NAME(ReleaseDetails)
 /**
  * The flag that indicates whether the release is a mandatory update or not.
  */
-@property(nonatomic, getter=isMandatoryUpdate, assign) BOOL mandatoryUpdate NS_SWIFT_NAME(mandatoryUpdate);
+@property(nonatomic, getter=isMandatoryUpdate, assign) BOOL mandatoryUpdate;
```
Comment (MatkovIvan) [inline AppCenterDistribute/AppCenterDistribute/Model/MSACReleaseDetails.h:41]: Same

Comment (DmitriyKirakosyan) [reply AppCenterDistribute/AppCenterDistribute/Model/MSACReleaseDetails.h:41]: It's not readonly, so it's `mandatoryUpdate` in Swift.

Comment (MatkovIvan) [reply AppCenterDistribute/AppCenterDistribute/Model/MSACReleaseDetails.h:41]: `NS_SWIFT_NAME` is required then

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,6 +4,7 @@
 
 ### App Center
 
+* **[Feature]** Make SDK API more Swift-friendly.
```
Comment (MatkovIvan) [inline CHANGELOG.md:7]: Breaking change? Add link to migration doc

File: CHANGELOG.md [hunk 4]
```diff
@@ -4,7 +4,7 @@
 
 ### App Center
 
-* **[Feature]** Make SDK API more Swift-friendly.
+* **[Feature]** **[Breaking change]** Make SDK API more Swift-friendly. You can find the API updates in the [swift migration table](replace-with-link).
```
Comment (MatkovIvan) [inline CHANGELOG.md:7]: I guess we already have the link

**Review by MatkovIvan — APPROVED** [review]
It's OK to merge it to the common feature branch

**Pull request merged**